src/System.Windows.Forms/tests/AccessibilityTests/Main.Designer.cs

```diff
@@ -147,6 +158,7 @@ private void InitializeComponent()
         private System.Windows.Forms.Button button6;
         private System.Windows.Forms.Button button7;
         private System.Windows.Forms.Button button8;
+        private System.Windows.Forms.Button button9;
```

Comment:
Strongly suggest to consider giving controls meaningful names, e.g. 
```suggestion
        private System.Windows.Forms.Button btnDataBindingExample;
```
or 
```suggestion
        private System.Windows.Forms.Button DataBindingExampleButton;
```
or whatever else you fancy.

By doing this you make the codebase significantly easier to read and reason about. That is, `button9_Click` makes no sense, whereas `btnDataBindingExample_Click` provides some usable information.